### ISSUE TITLE
Missing language attribute

### ISSUE THREAD

**Issue by jonnyry** [nhsconnect/gpconnect-demonstrator#344]

**Comment by jonnyry** [comment]
<html lang="en"> needs adding

**Comment by jonnyry** [comment]
Also fixed on the demonstrator wrapper pages... https://github.com/nhsconnect/gpconnect-demo-config/commit/72e1525302683a7e6829bc4e86f55d195a50e439
